src/get-styles.js

```diff
@@ -1,7 +1,7 @@
 function evaluateGlamorStyles(styles, props, theme, context) {
   return styles.map(style => {
     if (typeof style === 'function') {
-      return style(props, theme, context)
+      return style(Object.assign({}, props, {theme}), context)
```

Comment:
Might I suggest that we make this a backward compatible change for now by still passing the theme as the second argument? Them a breaking change can happen later.

Reply:
Totally agree. I should have just done that to start it. 

Is there a next branch I can show not the actual break to then?

Reply:
Just created a `next` branch @kwelch